src/testdir/test_cmdline.vim

```diff
@@ -1641,6 +1641,11 @@ func Test_getcmdtype()
   cunmap <F6>
 
   call assert_equal('', getcmdline())
+
+  call assert_equal('', getcmdmsg())
+  autocmd CmdlineChanged * call assert_equal('Answer?', getcmdmsg())
+  call feedkeys(":call input('Answer?')\<CR>", "t")
```

Comment:
The test seems not executed.
How to execute `input()` in tests?

Reply:
Change `"t"` to `"tx"`

Reply:
Fixed.